python/ray/tune/experimental/output.py

```diff
@@ -578,8 +591,15 @@ class ProgressReporter:
     _heartbeat_freq = 30  # every 30 sec
     # to be updated by subclasses.
     _heartbeat_threshold = None
+    _start_end_verbosity = None
+    _intermediate_result_verbosity = None
```

Comment:
Can the users specify these two verbosity independently, Or it's just for internal use?

Reply:
We haven't really worked on the configuration interface, yet, but these are just internal flags and shouldn't be specified.

Reply:
got it!